tests/product/test_collect.py

```diff
@@ -160,7 +163,10 @@ def get_query_id(self, sql, host=None):
         ips = self.cluster.get_ip_address_dict()
         if host is None:
             host = self.cluster.master
-        client = PrestoClient(ips[host], 'root')
+        config_path = os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)
+        config = self.cluster.exec_cmd_on_host(host, 'cat ' + config_path)
+        user = 'root'
+        client = PrestoClient(ips[host], user, PrestoConfig.from_file(StringIO(config), config_path, host))
```

Comment:
this solution works in product tests. Passing `host` and `user` is not enough for product tests as product test is not able to call things like (part of `coordinator_config()`):

```
            with settings(host_string='%s@%s' % (env.user, config_host)):
                with hide('stderr', 'stdout'):
                    temp_dir = run('mktemp -d /tmp/prestoadmin.XXXXXXXXXXXXXX')
```
fabric methods like `run` requires fabric to properly initialized, so it cannot be simply called from product tests